docs/resources/Guild.md

```diff
@@ -254,17 +254,17 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
-| roles          | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                                       |
-| joined_at      | ISO8601 timestamp                               | when the user joined the guild                                                                                                         |
-| premium_since? | ?ISO8601 timestamp                              | when the user started [boosting](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) the guild                |
-| deaf           | boolean                                         | whether the user is deafened in voice channels                                                                                         |
-| mute           | boolean                                         | whether the user is muted in voice channels                                                                                            |
-| pending?       | boolean                                         | whether the user has not yet passed the guild's [Membership Screening](#DOCS_RESOURCES_GUILD/membership-screening-object) requirements |
-| permissions?   | string                                          | total permissions of the member in the channel, including overrides, returned when in the interaction object                           |
+| Field         | Type                                            | Description                                                                                                                            |
+| ------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| user?         | [user](#DOCS_RESOURCES_USER/user-object) object | the user this guild member represents                                                                                                  |
+| nick?         | ?string                                         | this users guild nickname                                                                                                              |
+| roles         | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                                       |
+| joined_at     | ISO8601 timestamp                               | when the user joined the guild                                                                                                         |
+| premium_since | ?ISO8601 timestamp                              | when the user started [boosting](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) the guild                |
```

Comment:
was removing the optional marker from `premium_since` intentional here?

Reply:
https://github.com/discord/discord-api-docs/commit/364e7fb3fbf4b304778afcd503fc8a4100a7987d

Reply:
Doesn't take too long to get guild member update events without premiums_since though 😕